packages/ui/src/elements/ReactSelect/DropdownIndicator/index.tsx

```diff
@@ -0,0 +1,34 @@
+import type { DropdownIndicatorProps } from 'react-select'
+
+import { Chevron } from '@payloadcms/ui/icons/Chevron'
+import React from 'react'
+
+import type { Option as OptionType } from '../types.js'
+
+import './index.scss'
+
+const baseClass = 'dropdown-indicator'
+
+export const DropdownIndicator: React.FC<DropdownIndicatorProps<OptionType, true>> = (props) => {
+  const {
+    innerProps: { ref, ...restInnerProps },
+  } = props
+
+  return (
+    <div
+      className={baseClass}
+      ref={ref}
+      {...restInnerProps}
+      onKeyDown={(e) => {
+        if (e.key === 'Enter') {
+          // trigger the menu to open
+          return (e.key = ' ')
+        }
+      }}
+      role="button"
+      tabIndex={0}
+    >
+      <Chevron className={`${baseClass}__icon`} />
+    </div>
```

Comment:
I would rather the dropdown target element be a button than a div. See [here](https://github.com/payloadcms/payload/pull/6286/files#diff-ecc4ad56743dd2b308dbd9ce09258fde187ddc052f2d5d56c1fef55225eca855R16) on how to adjust the props type.